modules/virt-editing-vmis-web.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/cnv-manage-vmis.adoc
+
+[id="virt-editing-vmis-web_{context}"]
+= Editing a standalone virtual machine instance using the web console
+
+You can edit annotations and labels for a standalone virtual machine instance (VMI) using the web console. Other items displayed in the *Details* page for a standalone VMI are not editable.
```

Comment:
I have noticed that we have many topics in our cnv documentation where we do not use the preposition "by" in the sentence. Here is one example:
"Deleting the namespace using the web console"
https://docs.openshift.com/container-platform/4.4/cnv/cnv_install/uninstalling-container-native-virtualization.html#deleting-a-namespace-using-the-web-console_uninstalling-container-native-virtualization

I would like to understand when we should use "by" and when we should not use "by."
Since it is a suggestion, I am leaving it as it is for this release.

Regarding the second sentence - I think there are times when using passive voice is appropriate. For instance, if a series of sentences were all phrased in active voice, the end effect might make the writing look awkward! Can you please suggest how you would want the second sentence reworded in active voice? I tried rephrasing it in active voice and I felt the end result was awkward, therefore asking. Thanks!

Reply:
Again, in the absence of access to a paid online subscription to the Chicago Manual of Style, I requested Ingrid for an editing consultation on the second sentence. Her guidance was:

By second sentence, I think you mean: "Other items displayed in the *Details* page for a standalone VMI are not editable." Correct?

It is not currently in passive voice, and you can certainly keep it the way it is. It's fine. "are" is a linking verb and "editable" is an adjective modifying "items".

Reply:
With the guidance provided by Ingrid, I would prefer to keep the second sentence as it is without changes.

Reply:
Regarding using/by using: for the most part, we make exceptions for titles, but I was referring to the first sentence, not the title. Sorry for the confusion. The guidance comes from the IBM Style Guide (pg 25 in my version):

Use prepositions effectively to clarify the relationship between elements in a sentence. Follow these guidelines for using prepositions:
• Include prepositions that increase clarity.

Example (ambiguous)
Unload the file using the ULOAD utility.
 Example (clear)
Unload the file by using the ULOAD utility. 


Regarding the second sentence/active voice, I would prefer "You cannot edit..." But again, I don't think you really need this sentence. By saying "You can edit A and B" you're not implying that you can also edit the other things on the page. Just my 2c, take or leave. :)

Reply:
Since Ingrid has blessed that sentence and explained why it works fine with respect to active and passive voice, I like the idea of keeping it as it is!

Reply:
Another reason for keeping the second sentence (in my view), engineering liked to see that emphasis. And in the future, things will be different - other items in that Details view will be editable. Also in this release only annotations are editable in the Details page for VMIS, but other items are not. Which causes confusion for users (Matt Carleton agreed). Therefore, we are explicitly stating that other items in the Details tab are not.